Dockerfile

```diff
@@ -43,5 +49,3 @@ COPY --from=Builder --chown=1000:1000 $APP_HOME $APP_HOME
 WORKDIR $APP_HOME
 
 USER $APP_USER
-
-CMD bundle exec puma -C config/puma.rb
```

Comment:
So we don't need to run puma at all or we do it in compose?

Reply:
It runs in `docker-compose` and `container_definitions` for staging/production environment